frontend/pages/GameOver.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel ="stylesheet" href="../styles/Gameoverr.css">  
+    <h1></h1>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+</head>
+<body>
+  <div class="object">
+    <div class="shape">
+        <div class="GameOver">
+          <h1 id = "InnerText">GAME OVER</h1>
+          <div class="innerObject">
+            <div class="HighScores">
+              <div>
+                </div>
+                <ul id= "scores">
+                  <li>Score 1</li>
+                  <li>Score 2</li>
+                  <li>Score 3</li>
+                  <li>Score 4</li>
+                  <li>Score 5</li>
+                </div>
+              </ul>
+            <div class="innerShape"></div>
+          <div>
+          <button id ="Home" >Home</button>
+          <button id ="Retry" >Retry</button>
```

Comment:
We have the home and gameboard pages merged into main now, so its worth an attempt at linking to them here before merging this!